src/Runner.Worker/ExecutionContext.cs

```diff
@@ -137,6 +138,8 @@ public sealed class ExecutionContext : RunnerService, IExecutionContext
         public CancellationToken CancellationToken => _cancellationTokenSource.Token;
         public Dictionary<string, string> IntraActionState { get; private set; }
         public Dictionary<string, VariableValue> JobOutputs { get; private set; }
+
+        public ActionsEnvironmentReference ActionsEnvironment { get; private set; }
```

Comment:
do we need to add it to `CreateChild()` method?

Reply:
In future when we need environment context, we might refactor this, but for now, it's in jobcontext and is accessed from jobcontext in code.